lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -35,6 +58,12 @@ def on_def(node)
 
         private
 
+        def valid_method_name?(method_name, prefix)
```

Comment:
Since Ruby has its own notion about "valid" method names, perhaps we can use other nomenclature here? Maybe something like `#allowed_method_name?`